source/_integrations/modbus.markdown

```diff
@@ -688,6 +688,12 @@ climates:
         value 1 is written."
       required: false
       type: integer
+    hvac_onoff_register_write_type:
+      description: "Register (write) type of On/Off state. Valid options: `write_register`, `write_registers` and `write_coil`.
```

Comment:
@janiversen can you please explain how would this conflict?
In the code I made sure that if  `hvac_onoff_register_write_type` value is NOT set, the old defaults will be used ( where when write_registers is set,  `write_registers` is used, otherwise, write `register`)
This setting is only here in order to be override the defaults